metadata-ingestion/tests/integration/powerbi/golden_test_admin_only.json

```diff
@@ -1977,6 +1977,209 @@
     "aspectName": "dashboardInfo",
     "aspect": {
         "json": [
+            {
```

Comment:
Regarding these duplicates: I have tested with `patch_metadata: False` and there are no duplicates in that case; so there may be some issue with patch and/or some auto-processor.

Reply:
hm we should dig in a bit - duplicates might be masking a correctness problem

Reply:
Removing and creating golden files from scratch solved the issue of duplicated edges.
So you were true there may be an issue with update-golden-files when updating

Something to be solved, some day 😅